File: try/scores.py
from cs50 import get_int

scores = []
for i in range(3):
    scores.append(get_int("score: "))
    #---OR---
    score = get_int("score: ")
    scores += [score] #----OR---- scores = scores + [score]
average = sum(scores) / len(scores)
print(f"{average}")

File: try/scores.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n_sub = get_int("Number of subjects: ");
    int score[n_sub];

    for (int i = 0; i < n_sub; i++)
    {

        score[i] = get_int("score: ");
    }

    printf("Average: %.2f\n", (score[0] + score[1] + score[2]) / 3.0);
}